cmake/README.md

```diff
@@ -0,0 +1,28 @@
+## The CMake for Mbed OS
+
+Until further notice, CMake for Mbed OS is still in the development.  
+
+### Supported targets
+
+Only a limited set of targets is supported at the moment.
+
+These targets should work:
+- K64F 
+- DISCO_L475VG_IOT01A 
+- NRF52840_DK 
+- WIO_3G
+- K66F
+
+### Supported toolchains
+
+ARMClang6 and Gcc Arm toolchains are supported.
```

Comment:
Their official names are `Arm Compiler 6` and `GNU Arm Embedded`. Or if you want to use the names we use in Mbed you should probably say `ARM` and `GCC_ARM`.

Reply:
Let's use official names, I'll add them